packages/analytics-browser-test/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amplitude/analytics-browser-test",
-  "version": "2.7.0",
+  "version": "1.4.0-beta.0",
```

Comment:
This package started at 1.x and got a major version bump to 2.x when browser package got a version bump from 0.x to 1.x. Putting it on 1.4 since it is the first available one in 1.x.

This is a private package and is not published on npm.